smartfile/errors.py

```diff
@@ -33,7 +33,12 @@ def __init__(self, response, *args, **kwargs):
             if self.status_code == 400 and 'field_errors' in json:
                 self.detail = json['field_errors']
             else:
-                self.detail = json['detail']
+                try:
+                    # A faulty move request returns the below response
+                    self.detail = json['src'][0]
+                except KeyError:
+                    # A faulty delete request returns the below response
+                    self.detail = json['path'][0]
```

Comment:
Do we know that the json object will contain a key of `path` here? Because that could error out within the error?